docs/wip/demo_prometheus.md

```diff
@@ -0,0 +1,118 @@
+---
```

Comment:
When ready, I think moving this under a section called `Sample demo` in `metrics.md`  would be consistent with some of the forward-looking docs in the `tasks_usage` folder (egress, permissive mode, iptables etc.).

Reply:
Yeah, I wasn't exactly sure where this should go. The other spot I was thinking was in docs/demos/.